src/main/java/io/odpf/kafkaclient/consumer/ConsumerFactory.java

```diff
@@ -1,6 +1,14 @@
 package io.odpf.kafkaclient.consumer;
 
 import io.odpf.kafkaclient.ClientFactory;
+import io.odpf.kafkaclient.PropertiesNotSetException;
```

Comment:
What's the purpose of this Factory here dude. I think you could/should have skipped this.

Reply:
Since you had asked in previous review to create separate class to replace Producer and Consumer constructor code.